src/fal/planner/plan.py

```diff
@@ -14,7 +15,32 @@ class OriginGraph:
     graph: nx.DiGraph
 
     def copy_graph(self) -> nx.DiGraph:
-        return self.graph.copy()
+        return self.graph.copy()  # type: ignore
+
+    def _plot(self, graph=None):
+        """
+        For development and debugging purposes
+        """
+        if not graph:
+            graph = self.graph
+
+        import matplotlib.pyplot as plt
```

Comment:
Is this a dev dependency or does come automatically with networkx?

Reply:
Dev dependency. I will add it to pyproject.toml